code/modules/client/preferences_savefile.dm

```diff
@@ -138,6 +138,9 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 			S["hair_style_name"]	>> hairstyle
 		if(S["facial_style_name"])
 			S["facial_style_name"]	>> facial_hairstyle
+	if(current_version < 33)
+		randomise[RANDOM_HARDCORE] = FALSE
```

Comment:
No need to add this. A failed access to an associative list should return a falsey value already.

Reply:
alright, wasnt sure tbf working with prefs scares the fuck out of me and the same thing happens a few lines up so i figured better save than sorry